app/controllers/users/passwords_controller.rb

```diff
@@ -6,9 +6,15 @@ class Users::PasswordsController < Devise::PasswordsController
   layout "session"
 
   def create
-    super do |user|
-      if user.nil? || user.new_record?
-        flash.now[:notice] = ts("We couldn't find an account with that email address or username. Please try again?")
+    user_id = User.find_for_authentication(login: params[:user][:login])&.id
```

Comment:
Can you use `resource_params` as the argument?

- https://github.com/heartcombo/devise/blob/45b831c4ea5a35914037bd27fe88b76d7b3683a4/app/controllers/devise_controller.rb#L207-L209
- https://github.com/heartcombo/devise/blob/45b831c4ea5a35914037bd27fe88b76d7b3683a4/test/controllers/internal_helpers_test.rb#L38-L51